packages/manager/modules/pci/src/projects/project/sidebar/sidebar.constant.js

```diff
@@ -167,6 +171,15 @@ export const MENU = [
         title: 'Data Processing',
         regions: ['EU', 'CA'],
       },
+      {
+        id: 'logs-data-platform',
+        options: {
+          url: DBASS_LOGS,
+          external: false,
```

Comment:
```suggestion
          external: true,
``` 
maybe ?

Reply:
I am not sure... I thought since it is still a manager link (like the top nav bar), we do not have to have it as an external link... Others, please let us know what you think...

Reply:
As discussed with the team, we will leave it as internal. Thanks!